examples/policies/signed-by-github-actions.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Name: image-is-signed-by-github-actions
+#
+# Description:
+#   Assert that images are signed by a specific Github Actions
+#   workflow on the main branch.
+#
+
+apiVersion: policy.sigstore.dev/v1alpha1
+kind: ClusterImagePolicy
+metadata:
+  name: image-is-signed-by-github-actions
+spec:
+  images:
+  # All images in example repository matched
+  - glob: "**"
+  authorities:
+  - keyless:
+      # Signed by the public Fulcio certificate authority
+      url: https://fulcio.sigstore.dev
```

Comment:
Should we also add the ctlog pointing to public rekor?

Reply:
Is this required?

I hadnt added to other keyless examples: https://github.com/sigstore/policy-controller/blob/61447e886de6bc3ab373c5884824587827d3ecb0/examples/policies/keyless-attestation-sbom-spdxjson.yaml#L34

I'm wondering if we can merge this and add these later on? I've opened https://github.com/sigstore/policy-controller/issues/133